packit/source_git.py

```diff
@@ -422,35 +422,30 @@ def _put_downstream_sources(self, lookaside_sources: List[str]):
         self.local_project.stage(self.dist_git.source_git_downstream_suffix)
         self.local_project.commit(message="add downstream distribution sources")
 
-    def _get_lookasisde_sources(self) -> List[Dict[str, str]]:
+    def _get_lookaside_sources(self) -> List[Dict[str, str]]:
         """
         Read "sources" file from the dist-git repo and return a list of dicts
         with path and url to sources stored in the lookaside cache
         """
+        pkg_tool = "centpkg" if self.centos_package else "fedpkg"
```

Comment:
Hm :thinking: Probably it would be better to have a `--pkg-tool` option for `packit init`, too. Though I think it would be better to do this change as part of #1250.

Reply:
ok, I'll leave that up to you then